src/olympia/activity/tests/test_views.py

```diff
@@ -187,3 +195,65 @@ def _set_tested_url(self, pk=None, version_pk=None, addon_pk=None):
         self.url = reverse('version-reviewnotes-list', kwargs={
             'addon_pk': addon_pk or self.addon.pk,
             'version_pk': version_pk or self.version.pk})
+
+
+@override_settings(ALLOWED_CLIENTS_EMAIL_API=['10.10.10.10'])
+@override_settings(POSTFIX_AUTH_TOKEN='something')
+class TestEmailApi(TestCase):
+
+    def get_request(self, data=None, user=None):
```

Comment:
Why are those tests passing `user` ? Shouldn't it be completely useless, since no authentication is performed by the API ?